onnx/version_converter/adapters/maxpool_17_18.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Adapter for MaxPool in default domain from version 17 to 18
+
+#pragma once
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE {
+namespace version_conversion {
+
+class MaxPool_17_18 final : public Adapter {
```

Comment:
No need for a specialized adapter. Please see comment below.

Reply:
Done.